openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,35 +60,38 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A set of one or more claims about a subject, provided by a Credential Issuer.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Credential (or Verifiable Credential):
```

Comment:
```suggestion
Credential (or Verifiable Credential, or Credential Instance):
```

Reply:
I don't think it is a good idea to introduce "X Instance" when we already have "X". I see what you want to clarify with this term, but having both meaning the same doesn't make sense. And using "Credential Instance" everywhere doesn't seem like a good idea either ("Batch Credential Instance Issuance Endpoint?").

Reply:
I concur with @danielfett

Reply:
I concur with @awoie who concurs with @danielfett

Reply:
I got convinced in the wg call, happy to close this

Reply:
WG call, agreed not to make this change.